docs/modules/ROOT/pages/performance/sql_and_scan_query_pref_comp.adoc

```diff
@@ -33,17 +33,17 @@ instantiation, cache.withKeepBinary().query(...)
 SCAN_CURSOR_FETCH time represents time required to extract the result from
 result set (cursor, cursor.getAll() API).
 
-For Insert operation, MAX represents total time to insert 1M entires into the cache and AVG represents the average time to insert one single entry. All other statistcs for Insert are not relevant and are shown as zero.
+For Insert operation, MAX represents total time to insert 1M entires into the cache and AVG represents the average time to insert one single entry. All other statistcs for Insert are not relevant and are set to zero.
```

Comment:
If the data is unavailable, mark the entry as N/A or leave it vacant. 

Setting to zero is very confusing.

Reply:
Changed as noted. Showing 'N/A' instead.